protocols/luci-proto-hnet/Makefile

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (C) 2018 Florian Eckert <fe@dev.tdt.de>
+#
+# This is free software, licensed under the Apache License, Version 2.0 .
+#
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=Support for hnet
+LUCI_DEPENDS:=
```

Comment:
There are 3 packages selectable hnet-full, hnet-full-secure or hnet-full-l2tp I am not sure how to depend on a package with variants.
I think we need a extra package for the hnet netifd proto?
If so I will slice the netifd proto out of the variants https://github.com/openwrt-routing/packages/blob/master/hnetd/Makefile
and add a new package hnet which has only the proto of the netifd.

Reply:
I see, seems the same question was also asked there in the comment ;)  I may resort to `PROVIDES` and try `depends` instead of `select`.  I think it's okay we leave it be and let LuCI test the existence of hnet.sh

Reply:
I have staged a PR on my github acount to move the hnet proto for netifd to an own package.
So we could depend on hnet?
https://github.com/TDT-AG/packages-1/commit/6100b458e9e615fe39d96b7f4cbdbe9bb8fb73d6

Reply:
It's a bit overkill for the hnet package, plus I am afraid having only hnet.sh is not sufficient for the protocol to work in the first place.